cblox_msgs/package.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>cblox_msgs</name>
+  <version>0.0.0</version>
+  <description>Message definitions for Cblox ROS interface.</description>
+
+  <author email="helen.oleynikova@mavt.ethz.ch">Helen Oleynikova</author>
+  <author email="marius.fehr@mavt.ethz.ch">Marius Fehr</author>
+  <author email="gasserl@student.ethz.ch">Laura Gasser</author>
+
+  <maintainer email="helen.oleynikova@mavt.ethz.ch">Helen Oleynikova</maintainer>
+  <maintainer email="marius.fehr@mavt.ethz.ch">Marius Fehr</maintainer>
```

Comment:
I guess let's change this to you and I? Not that anyone has probably ever even looked at these definitions.